scripts/aws_clear_instance.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+echo "Removing application and dependencies"
+
+if [ -d "/home/notify-app/notifications-api" ]; then
+    # Remove and re-create the directory
+    rm -rf /home/notify-app/notifications-api
+    mkdir -vp /home/notify-app/notifications-api
+    # Remove installed py3 packages
+    pip3 freeze | xargs pip3 uninstall -y
```

Comment:
@bandesz it will cause version issues

if i've got a wheelhouse with requests == 2.3 and requests == 2.4 and say

```
(foo) [leohemsted:~]$ pip install --use-wheel --no-index --find-links=wheelhouse wheelhouse/*
Double requirement given: requests==2.4.0 from file:///Users/leohemsted/wheelhouse/requests-2.4.0-py2.py3-none-any.whl (already in requests==2.3.0 from file:///Users/leohemsted/wheelhouse/requests-2.3.0-py2.py3-none-any.whl, name='requests')
```